source/constructs/lib/back-end/back-end-construct.ts

```diff
@@ -35,6 +34,7 @@ export interface BackEndProps extends SolutionConstructProps {
   readonly logsBucket: IBucket;
   readonly uuid: string;
   readonly cloudFrontPriceClass: string;
+  readonly s3KmsKeyArn: string
```

Comment:
Since our s3 image buckets use KMS encryption, had to pass a new param for the key ARN, so that we can give decrypt permissions to the cloudformation user.